### PR TITLE
Feature/tiny-mce

### PR DESCRIPTION
**Pull request by J-pilon**

### CODE REVIEW

File: app/javascript/components/articles/NewPage.jsx [hunk 1]
```diff
@@ -15,23 +17,40 @@ const NewPage = (props) => {
...
 				</Label>
 
 				<Label>
 					Enter the articles body:
 
-					<Input
-						placeholder="body"
-						name="article[body]"
-						id="article_body"
+					<Editor
```
Comment (jensen) [inline app/javascript/components/articles/NewPage.jsx:36]: Editor for create and Editor for edit. Both are very similar and have basically been copy/pasted. We can create components so we should. There is no way this should be defined twice when it is so similar.

File: app/javascript/components/articles/NewPage.jsx [hunk 2]
```diff
@@ -15,23 +17,40 @@ const NewPage = (props) => {
...
+						apiKey="82i1zra2emdhclzhbkp2fdtnf0oafyx5yxr89bkls8cqaaur"
```
Comment (jensen) [inline app/javascript/components/articles/NewPage.jsx:37]: Did you want to embed your API key like this? Even if it's a client API key it should be using an environment variable so you can change it depending on deployment.

File: app/javascript/components/articles/EditPage.jsx [hunk 3]
```diff
@@ -39,14 +41,34 @@ const EditPage = (props) => {
...
 
 					<Input
 						name="article[body]"
 						defaultValue={props.article.body}
 					/>
-				</Label>
+				</Label> */}
+
+				<Editor
+					apiKey="82i1zra2emdhclzhbkp2fdtnf0oafyx5yxr89bkls8cqaaur"
```
Comment (aarondufall) [inline app/javascript/components/articles/EditPage.jsx:54]: Do you want this API key in your code and more importantly in your version control?

File: app/javascript/components/articles/EditPage.jsx [hunk 4]
```diff
@@ -39,14 +41,34 @@ const EditPage = (props) => {
...
+						'advlist autolink lists link image charmap print preview anchor',
+						'searchreplace visualblocks code fullscreen',
+						'insertdatetime media table paste code help wordcount'
+					],
+					toolbar: 'undo redo | formatselect | ' +
+					'bold italic backcolor | alignleft aligncenter ' +
+					'alignright alignjustify | bullist numlist outdent indent | ' +
+					'removeformat | help',
+					content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+					}}
```
Comment (aarondufall) [inline app/javascript/components/articles/EditPage.jsx:70]: Do you want all this configuration in the component itself or could it be imported from a separate file?

File: app/javascript/components/articles/NewPage.jsx (hunk 1 above)
Comment (J-pilon) [reply app/javascript/components/articles/NewPage.jsx:36]: smh! yep agreed.

File: app/javascript/components/articles/EditPage.jsx (hunk 3 above)
Comment (J-pilon) [reply app/javascript/components/articles/EditPage.jsx:54]: I'm an idiot lol. No I don't want that to be public.

File: app/javascript/components/articles/EditPage.jsx [hunk 5]
```diff
@@ -39,14 +41,34 @@ const EditPage = (props) => {
...
-				<Label>
+				{/* <Label>
 					Body
 
 					<Input
 						name="article[body]"
 						defaultValue={props.article.body}
 					/>
-				</Label>
+				</Label> */}
```
[the remark this answers is not in the document]
Comment (J-pilon) [reply app/javascript/components/articles/EditPage.jsx:51]: Yes, it is still work in progress. I should have titled the PR with "wip".

File: app/javascript/components/articles/EditPage.jsx (hunk 4 above)
Comment (J-pilon) [reply app/javascript/components/articles/EditPage.jsx:70]: Ok, not sure exactly what that would look like but I have an idea and i'll play around with it first.

**Pull request merged**